src/arrays/creating_arrays/hard/q2/MainTest.java

```diff
@@ -0,0 +1,95 @@
+package arrays.creating_arrays.hard.q2;
+
+import global.BaseTest;
+import global.tools.CustomAssertions;
+import global.tools.TestOption;
+import global.utils.MethodUtil;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Java
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "39 67 123 5 21";
+        return new Clause[]{
```

Comment:
We may want to think about using a `ClauseBuilder()` here so that we can more easily vary the input on the website.

Reply:
It would also just make the code cleaner